pandas/core/series.py

```diff
@@ -2989,12 +2989,15 @@ def combine(self, other, func, fill_value=None) -> Series:
 
     def combine_first(self, other) -> Series:
         """
-        Combine Series values, choosing the calling Series's values first.
+        Update null elements with value in the same location in 'other'.
+
+        Combine two Series objects by filling null values in one Series with
+        non-null values from the other Series.
```

Comment:
per the corresponding line in `frame.py`, perhaps mention that the resulting index will be the union of the two?

Reply:
It was in the "Notes" section, I moved that for consistency.